packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3802,281 +3906,292 @@ describe('plugin-meetings', () => {
             sendShare: false,
           };
           meeting.isMultistream = true;
-          meeting.mediaProperties.webrtcMediaConnection = {
-            publishTrack: sinon.stub().resolves({}),
-            unpublishTrack: sinon.stub().resolves({}),
-          };
-          meeting.audio = { handleLocalTrackChange: sinon.stub()};
-          meeting.video = { handleLocalTrackChange: sinon.stub()};
-
-          const createFakeLocalTrack = (originalTrack) => ({
-            on: sinon.stub(),
-            off: sinon.stub(),
-            stop: sinon.stub(),
-            originalTrack,
+          meeting.mediaProperties.webrtcMediaConnection = {};
+          meeting.audio = { handleLocalStreamChange: sinon.stub()};
+          meeting.video = { handleLocalStreamChange: sinon.stub()};
+          meeting.sendSlotManager = new SendSlotManager({
+            logger: {
+              info: () => undefined
+            }
           });
+          fakeMultistreamRoapMediaConnection = {
+            createSendSlot: sinon.stub().returns({
+              publishStream: sinon.stub(),
+              unpublishStream: sinon.stub(),
+            })
+          }
+          meeting.sendSlotManager.createSlot(fakeMultistreamRoapMediaConnection, MediaType.VideoSlides);
```

Comment:
this doesn't feel right..... in all the unit tests in this file we're using same `meeting` object and creating slots here will impact all other tests. we should probably use sinon to just mock the right wcme send slots and then restore those mocks or if this turns out to be difficult than at least undo the creation of the slot in `afterEach()`

Reply:
Agreed. Added `afterEach` routine with each `createSendSlot` in `beforeEach`

Reply:
actually I think I was wrong, because the meeting object is created in the beforeEach() so a new one will be created for each test, so these sendSlots won't affect other tests.